src/extern/cef/cef_drag_data.pxd

```diff
@@ -8,8 +8,13 @@ from cef_ptr cimport CefRefPtr
 from cef_image cimport CefImage
 from cef_types cimport CefPoint
 
+from libcpp.vector cimport vector as cpp_vector
+
 cdef extern from "include/cef_drag_data.h":
     cdef cppclass CefDragData:
+        cpp_bool IsFile()
```

Comment:
If the methods were ordered alphabetically before then please keep an alphabetic order when adding new ones.